app/globals.css

```diff
@@ -91,4 +88,7 @@ body {
   .flex-start {
     @apply flex justify-start items-center;
   }
+  .auth-background {
+    @apply bg-black-900 min-w-full min-h-full;
+  }
```

Comment:
There are two of these. Remove the one in theme.css or remove the one in global.css

We should just have one- and it could be:

```js
  .auth-background {
    @apply h-screen bg-black-900 flex flex-col items-center justify-start gap-y-40 pt-[72px];
  }
```
To control much of our layout. Like before, this could probably be renamed to `auth-page` or `auth-layout` and it would make more sense.